pkg/controller/storagecluster/cephcluster.go

```diff
@@ -485,6 +490,8 @@ func newStorageClassDeviceSets(sc *ocsv1.StorageCluster, serverVersion *version.
 				Encrypted:            sc.Spec.Encryption.Enable,
 			}
 
+			ds.DataPVCTemplate.Annotations = annotations
+			set.VolumeClaimTemplates = append(set.VolumeClaimTemplates, ds.DataPVCTemplate)
```

Comment:
Doesn't this mean we'll have two copies of `ds.DataPVCTemplate` in set?
Wouldn't it be better to first set the annotations into `ds.DataPVCTemplate.Annotation` and then define `set`?

Reply:
added the annotations  before set.

Reply:
thanks, much better!